indicnlp/tokenize/sentence_tokenize.py

```diff
@@ -216,6 +230,12 @@ def sentence_split(text,lang,delim_pat='auto'): ## New signature
         ## NEW
         if p1>0 and text[p1-1].isnumeric():
             continue
+        
+        ## Prevents splitting on "." in URLs/emails in indic texts.
+        if lang != "en":
+            if is_latin_or_numeric(text[p1-1]):
+                if p1+1 < len(text) and is_latin_or_numeric(text[p1+1]):
```

Comment:
The assumption here is that the text is well-formed. If this were truly a sentence delimiter, there would be a space or new-line, etc. after the sentence delimiter. This assumption is fine for clean text - but what about something like 'This is the first sentence.This is second sentence.' - this would not be sentence tokenized.

Reply:
ok, your argument that this boundary case applies for latin scripts only where we find URLs/emails is plausible. Sounds good.